src/Results/DTO/Candidate.php

```diff
@@ -101,18 +110,52 @@ public static function getJsonSchema(): array
         return [
             'type' => 'object',
             'properties' => [
-                'message' => Message::getJsonSchema(),
-                'finishReason' => [
+                self::KEY_MESSAGE => Message::getJsonSchema(),
+                self::KEY_FINISH_REASON => [
                     'type' => 'string',
                     'enum' => FinishReasonEnum::getValues(),
                     'description' => 'The reason generation stopped.',
                 ],
-                'tokenCount' => [
+                self::KEY_TOKEN_COUNT => [
                     'type' => 'integer',
                     'description' => 'The number of tokens in this candidate.',
                 ],
             ],
-            'required' => ['message', 'finishReason', 'tokenCount'],
+            'required' => [self::KEY_MESSAGE, self::KEY_FINISH_REASON, self::KEY_TOKEN_COUNT],
```

Comment:
I'm not sure every provider returns a `token_count` on an individual candidate? Should we leave that optional? Or would we prefer consistency and either set to `0` or try some estimation if it's not provided?

Reply:
Ohh, good question. How would token count be used downstream? Would folks be more likely to prefer to have something? Or would it be important to know that the  model didn't provide one? The use case isn't clear enough for me to make a suggestion. 🤔

Reply:
I'm not sure yet myself 😆 
Let's leave this for a later discussion, if it becomes more relevant.